frontend/sozisel-app/src/components/PresenterSession/PresenterSession.tsx

```diff
@@ -0,0 +1,47 @@
+import "./PresenterSession.scss";
+
+import { Grid, Paper } from "@material-ui/core";
+import React, { ReactElement } from "react";
+
+import Agenda from "./Agenda/Agenda";
+import MainNavbar from "../Navbar/MainNavbar/MainNavbar";
+import { useParams } from "react-router";
+import { useSessionDetailsQuery } from "../../graphql";
+
+export default function PresenterSession(): ReactElement {
+  const { id } = useParams<{ id: string }>();
+
+  const { data, loading } = useSessionDetailsQuery({
+    fetchPolicy: "network-only",
```

Comment:
Is there any reason for `network-only`?

Reply:
Not really, I just copy-pasted this query from another file. I will remove this fetchPolicy.